app/support/page.tsx

```diff
@@ -0,0 +1,72 @@
+'use client'
+import Link from 'next/link';
+import React from 'react';
+
+const handleSave = (e: { preventDefault: () => void; }) =>{
+  e.preventDefault()
+  console.log("save button clicked!")
+}
```

Comment:
if you're up for it, a good optimization to apply for buttons are throttling. 

Throttling ensures that a function is executed at most once in a specified period of time, regardless of how many times the button is clicked. In a Next.js app, you can implement throttling for a button using utilities like lodash.throttle or a custom function.

a popular library for optimization utilities is `lodash`

you can `npm install lodash`

then add throttling to button click as follow

```
import { throttle } from "lodash";

export default function ThrottledButton() {
  const handleClick = () => {
    console.log("Button clicked");
    // Your logic here
  };

  // Throttle the handler to execute at most once every 2 seconds
  const throttledClickHandler = throttle(handleClick, 2000);

  return (
    <button
      onClick={throttledClickHandler}
    >
      Throttled Button
    </button>
  );
}
```